wiki/machine-learning/python-libraries-for-reinforcement-learning.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2020-12-07
+title: Python libraries for Reinforcement Learning
+---
+[Reinforcement Learning (RL)](https://en.wikipedia.org/wiki/Reinforcement_learning)is a machine learning approach for teaching agents how to solve tasks by trial and error. More specifically, RL is mostly concerned with how software agents should take actions in an environment in order to maximize its cumulative reward. The application of RL, as it seeks a solution to balance exploration and exploitation, ranges from Resource Management, Traffic Light Control, Recommendation, and Advertising, to Robotics. A survey on RL with Robotics could be found [here](https://www.ias.informatik.tu-darmstadt.de/uploads/Publications/Kober_IJRR_2013.pdf). The successes of deep learning and reinforcement learning area in recent years have led many researchers to develop methods to control robots using RL with the motivation to automate the process of designing sensing, planning, and control algorithms by letting the robot learn them autonomously. This post gives a brief introduction to a few popular RL libraries in the Robotics context, from beginning to immediate, to advanced level users. At last, we will provide general tips for a more in-depth study on RL topics and link to a concrete example of using RL to formulate a self-driving agent.
```

Comment:
The links in the header here won't display properly when the summaries get generated. 

From the template: `The first 100 words are used as an excerpt on the Wiki's Index. No images, HTML, or special formating should be used in this section as it won't be displayed properly.`

They should be moved to References or below the first subheading (i.e. "Spinning Up")

Reply:
Thanks and I moved them to See Also links!